tests/integration/server.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/python
+
+# Copyright 2016 Google Inc. All rights reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+import google.cloud.logging
+import google.cloud.monitoring
+import google.cloud.error_reporting
+import google.cloud.exceptions
+
+from flask import Flask, request, jsonify
+
+app = Flask(__name__)
+
+
+@app.route('/')
+def hello_world():
+    return 'Hello World!'
+
+
+@app.route('/logging', methods=['POST'])
+def _logging():
+    request_data = request.get_json()
+    if request_data is None:
+        raise ErrorResponse('Unable to parse request JSON: '
+                            'did you set the Content-type header?')
+    log_name = request_data.get('log_name')
+    if not log_name:
+        raise ErrorResponse('Please provide log name')
+    token = request_data.get('token')
+    if not token:
+        raise ErrorResponse('Please provide token name')
+
+    _log(token, log_name)
+
+    return 'OK', 200
+
+
+def _log(token, log_name='stdout'):
+    """
+    Write a log entry to Stackdriver.
+
+    Keyword arguments:
+    token -- 16-character (8-byte) hexadecimal token, to be written
+    as a log entry.
+    log_name -- The name of the logging group to be written to.
+
+    Once the entry is written to Stackdriver, the test driver will retrieve
+    all entries with the name 'log_name', and verify there is an entry with
+    the same value as 'token', indicating the entry was written successfully.
+    """
+
+    # TODO (nkubala): just as a note, currently the client logging API is
+    # broken
+
+    # TODO (nkubala): write token to 'log_name' log, instead of stdout
+    # is this possible in non-standard (flex)???
+
+    try:
+        client = google.cloud.logging.Client()
+        gcloud_logger = client.logger(log_name)
+        gcloud_logger.log_text(token)
+    except Exception as e:
```

Comment:
catch `google.cloud.exceptions.GoogleCloudError` instead of this broad exception?